makes people spawn with noslips on BR

ZWRh3 opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- To automatically tag this PR, add the uppercase label(s) surrounded by brackets below, for example: [BALANCE] -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
makes players on the BR gamemode start with noslips shoes
## Why's this needed? <!-- Describe why you think this should be added to the game. -->
i have seen a ton of BR games turn into slip hell in pools of blood when people try to travel trough the halls at above walk speed, this is very bad given that you have merely 60 seconds to get to a safezone, this also changes the combat meta a bit but i doubt the change to it is major

also removes: galoshes, HOS boots, banana nades from the weapon/armor item pools due to being useless
HOS shoes replaced by nukie knight shoes in its pool

TobleroneSwordfish commented:
Bear in mind that this completely invalidates banana grenades and possibly a few other items that should be removed from the BR pool if this passes

ZWRh3 commented:
> Bear in mind that this completely invalidates banana grenades and possibly a few other items that should be removed from the BR pool if this passes

removed those and replaced HOS shoes
